topics/geode-mods/index.md

```diff
@@ -3,7 +3,8 @@ created_by: The Bearodactyl
 display_name: Geode Mods
 short_description: This topic is for people to advertise Geode mods.
```

Comment:
Usually the short description has a little context about the topic - how about:
```suggestion
short_description: Geode is a modding framework for the rhythm platformer game Geometry Dash
```

Reply:
yeah, i could do that :D